fedora/jellyfin-selinux-launcher.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec /usr/lib64/jellyfin/jellyfin ${@}
```

Comment:
This relies on the assumption that `%{_libdir}` is always going to be `/usr/lib64`.  A more portable and future-proof method would be to substitute `%{_libdir}` into this script before installing it.